src/v/datalake/translation/partition_translator.cc

```diff
@@ -185,7 +186,9 @@ ss::future<> partition_translator::translate_until_stopped() {
     bool needs_jitter = false;
     while (!_as.abort_requested()) {
         if (needs_jitter) {
-            co_await ss::sleep_abortable(_jitter.next_duration(), _as);
+            auto sleep_dur = _jitter.next_jitter_duration();
+            vlog(_logger.trace, "Sleeping for {}ms", sleep_dur);
```

Comment:
nit: do we need that log entry, is is helpful i am wondering as this will be logged per partition